Yank/tests/test_analyze.py

```diff
@@ -182,7 +182,7 @@ def test_repex_phase_initilize(self):
     def test_repex_mixing_stats(self):
         """Test that the Repex Phase yields mixing stats that make sense"""
         phase = analyze.ReplicaExchangeAnalyzer(self.reporter, name=self.repex_name)
-        t, mu = phase.generate_mixing_statistics()
+        t, mu, g = phase.generate_mixing_statistics()
```

Comment:
Not something you have to but remember if you don't use a variable you expand from, you can always cast it to `_` to get the IDE to stop complaining about unused variable